recipes/pcapplusplus/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "PcapPlusPlus" %}
+{% set version = "22.05" %}
+
+package:
+  name: lib{{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/seladb/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5f299c4503bf5d3c29f82b8d876a19be7dea29c2aadcb52f2f3b394846c21da9
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
+  - name: libpcapplusplus
+    build:
+      run_exports:
+       - {{ pin_subpackage( 'pcapplusplus-licenses', max_pin='x.x') }}
+    script: build-lib.sh
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - make
+      host:
+        - libpcap
+    test:
+      commands:
+        - test -f ${PREFIX}/include/pcapplusplus/PcapPlusPlusVersion.h
+        - test -f ${PREFIX}/lib/libCommon++.a
+        - test -f ${PREFIX}/lib/libPacket++.a
+        - test -f ${PREFIX}/lib/libPcap++.a
+  - name: pcapplusplus-licenses
+    build:
+      noarch: generic
+
+about:
+  home: https://pcapplusplus.github.io/
+  license: Unlicense
```

Comment:
Use the `AND` keyword to list multiple license types.

https://conda-forge.org/docs/maintainer/adding_pkgs.html#about